src/translations.js

```diff
@@ -106,6 +109,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       sk: "Všetky PR",
       tr: "Toplam PR",
       pl: "Wszystkie PR",
+      uz: "'Pull Request'lar",
```

Comment:
I'm very curious why the `'` is used around `pull requests`, `commits` and `isues` in your language. 🤓

Reply:
I couldn't find the exact translations.
In our country (Uzbeks) these words are mainly used in the English language :grimacing:

Reply:
Ah makes sense thanks!